README.adoc

```diff
@@ -45,18 +45,19 @@ image::reactive-messaging-system-inventory.png[Reactive system inventory applica
 *True-to-production integration testing with MicroShed Testing*
 
 Tests sometimes pass during the development and testing stages of an application's lifecycle but then fail in production
-because the environment in which the application is running is different. While you can create mock objects and custom
-setups to try to minimize differences between environments, it is difficult to mimic a production system for an application
-that uses an external messaging system like Kafka. MicroShed Testing addresses this problem by enabling you to test your
-application in the same Docker container, or containers, that you'll use in production. This means that the environment
-remains the same throughout the application's lifecycle, from development, through testing, and into production.
+because of differences between your development and production environments. While you can create mock objects and custom
+setups to minimize differences between environments, it is difficult to mimic a production system for an application
+that uses an external messaging system. MicroShed Testing addresses this problem by enabling the testing of applications
+in the same Docker containers that you’ll use in production. As a result, your environment remains the same throughout
+the application’s lifecycle – from development, through testing, and into production.
 You can learn more about MicroShed Testing in the
 https://openliberty.io/guides/microshed-testing.html[Testing a MicroProfile or Jakarta EE application^] guide.
 
 == Additional prerequisites
 
-Install Docker (https://docs.docker.com/engine/installation[see the official Docker documentation^]) and start your Docker environment.
-You will build and run the microservices in Docker containers. An installation of Apache Kafka is provided in another Docker container.
+You need to have Docker installed. For installation instructions, refer to the
+(https://docs.docker.com/engine/installation[official Docker documentation^]) You will build and run the microservices in
```

Comment:
I'd suggest updating 
`For installation instructions, refer to the (https://docs.docker.com/engine/installation[official Docker documentation^])`
To
`For installation instructions, refer to the official link:https://docs.docker.com/get-docker/[Docker documentation].`